### PR TITLE
add Project file and update CI scripts

### PR DESCRIPTION
**Pull request by KristofferC**
Replaces #1440

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -36,10 +37,19 @@ install:
   - C:\projects\julia-binary.exe /S /D=C:\projects\julia
 
 build_script:
-# Need to convert from shallow to complete for Pkg.clone to work
-  - IF EXIST .git\shallow (git fetch --unshallow)
-  - C:\projects\julia\bin\julia -e "versioninfo();
-      Pkg.clone(pwd(), \"DataFrames\"); Pkg.build(\"DataFrames\")"
+  - C:\projects\julia\bin\julia -e "
```
Comment (quinnj) [inline appveyor.yml]: @simonbyrne, does your new AppVeyor.jl script take this 0.7 version check into account automatically? Should we just switch to that for DataFrames here?

Comment (simonbyrne) [reply appveyor.yml]: Yes it does.

**Comment by nalimilan** [comment]
@KristofferC Sorry for not merging this earlier. Do you think the CI error is legitimate?

**Comment by quinnj** [comment]
Already switched over to Project.toml

**Pull request closed without merging**